qb/select.go

```diff
@@ -115,10 +115,9 @@ func (b *SelectBuilder) Columns(columns ...string) *SelectBuilder {
 	return b
 }
 
-// ColumnAs adds a result column with a custom name to the query.
-func (b *SelectBuilder) ColumnAs(column, name string) *SelectBuilder {
-	b.columns = append(b.columns, column+" AS "+name)
-	return b
+// As is a helper for adding a column AS name result column to the query.
```

Comment:
Drop the previous commit, move the function it to `utils.go` and add a smaller unit test there.